fastlane-plugin-wpmreleasetoolkit.gemspec

```diff
@@ -56,4 +61,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'simplecov', '~> 0.16.1'
   spec.add_development_dependency 'fastlane', '~> 2'
   spec.add_development_dependency 'rmagick', '~> 4.1'
+  spec.add_development_dependency 'cocoapods', '~> 1.10'
```

Comment:
My rationale is that, by adding this as a development dependency only, we won't be able to bump dependencies to versions that are not compatible with CocoaPods locally. At the same time, by being only a _development_ dependency, users of this gem won't need to install CocoaPods, which I think is valuable for the Android repos which are still using the Ruby toolchain.